README.md

```diff
@@ -88,10 +88,10 @@ import Heapify from "https://unpkg.com/heapify@0.2.1"
 import Heapify from "heapify";
 
 const queue = new Heapify();
-queue.push(1, 10);
-queue.push(2, 5);
-queue.pop();  // 2
-queue.peek();  // 1
+queue.push('foo', 10); // add 'foo' with priority 10
+queue.push('bar', 5); // add 'bar' with priority 5
+queue.pop();  // 'bar'
+queue.peek();  // 'foo'
```

Comment:
Would this work? Since it uses typed arrays and can't store strings. It just stores `0`s as keys for me